webpack.config.js

```diff
@@ -59,32 +59,22 @@ export default {
     new CopyWebpackPlugin({
       patterns: [
         {
-          from: 'schemas/fingerprint-server-api.yaml',
-          to: 'schemas/fingerprint-server-api.yaml',
```

Comment:
I think we should keep this schema, we mention it in the documentation and customers could relay on it

Reply:
Fixed